Formula/hound.rb

```diff
@@ -0,0 +1,17 @@
+require "formula"
+
+class Hound < Formula
+  homepage "https://github.com/thoughtbot/hound-cli"
+  url "https://github.com/thoughtbot/hound-cli/archive/v0.1.0.tar.gz"
+  version "0.1.0"
```

Comment:
I've always defined it explicitly (and once did so in error, mismatching versions), but it does look like [Homebrew can infer it from the URL](github.com/github/hub/Godeps/_workspace/src/github.com/octokit/go-octokit/octokit). We might want to give this a try.